strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/proxied/SimpleProxyRepositoryArtifactResolver.java

```diff
@@ -27,16 +35,27 @@
 
     private static final Logger logger = LoggerFactory.getLogger(SimpleProxyRepositoryArtifactResolver.class);
 
+    @Inject
+    private ArtifactByteStreamsCopyStrategyDeterminator artifactByteStreamsCopyStrategyDeterminator;
+
     @Override
     protected InputStream onSuccessfulProxyRepositoryResponse(InputStream is,
                                                               String storageId,
                                                               String repositoryId,
                                                               String path)
             throws IOException, NoSuchAlgorithmException, ProviderImplementationException
     {
-        final java.io.File file = java.io.File.createTempFile("strongbox", ".tmp");
-        FileUtils.copyInputStreamToFile(is, file);
-        return Files.newInputStream(file.toPath());
+        final Path tempPath = Files.createTempFile("strongbox", ".tmp");
```

Comment:
I have only just noticed this. It's incorrect. You can get the path for this the following way:
`ConfigurationResourceResolver.getHomeDirectory()`. You can also check `ConfigurationResourceResolver.getConfigurationResource(...)`.

Reply:
We should actually add some new methods to it like... for example, `getEtcDirectory()`. `getTempDirectory()` and so on.

Reply:
Done